specification/security/resource-manager/Microsoft.Security/preview/2019-01-01-preview/subAssessments.json

```diff
@@ -0,0 +1,604 @@
+					},
+					{
+						"$ref": "#/parameters/ODataExpand"
+					}
+				],
+				"responses": {
+					"200": {
+						"description": "OK",
+						"schema": {
+							"$ref": "#/definitions/SecuritySubAssessmentList"
+						}
+					},
+					"default": {
+						"description": "Error response describing why the operation failed.",
+						"schema": {
+							"$ref": "../../../common/v1/types.json#/definitions/CloudError"
+						}
+					}
+				},
+				"x-ms-pageable": {
+					"nextLinkName": "nextLink"
+				}
+			}
+		},
+		"/{resourceId}/providers/Microsoft.Security/assessments/{assessmentName}/subAssessments/{subAssessmentName}": {
+			"get": {
+				"x-ms-examples": {
+					"Get security recommendation task from security data location": {
+						"$ref": "./examples/SubAssessments/GetSubAssessment_example.json"
+					}
+				},
+				"tags": ["SubAssessments"],
+				"description": "Get a security sub-assessment on your scanned resource",
+				"operationId": "SubAssessments_Get",
+				"parameters": [{
+					"$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+				},
+				{
+					"$ref": "../../../common/v1/types.json#/parameters/ResourceId"
+				},
+				{
+					"$ref": "#/parameters/AssessmentName"
+				},
+				{
+					"$ref": "#/parameters/SubAssessmentName"
+				}],
+				"responses": {
+					"200": {
+						"description": "OK",
+						"schema": {
+							"$ref": "#/definitions/SecuritySubAssessment"
+						}
+					},
+					"default": {
+						"description": "Error response describing why the operation failed.",
+						"schema": {
+							"$ref": "../../../common/v1/types.json#/definitions/CloudError"
+						}
+					}
+				}
+			}
+		}
+	},
+	"definitions": {
+		"SecuritySubAssessmentList": {
+			"type": "object",
+			"description": "List of security sub-assessments",
+			"properties": {
+				"value": {
+					"readOnly": true,
+					"type": "array",
+					"items": {
+						"$ref": "#/definitions/SecuritySubAssessment"
+					}
+				},
+				"nextLink": {
+					"readOnly": true,
+					"type": "string",
+					"description": "The URI to fetch the next page."
+				}
+			}
+		},
+		"SecuritySubAssessment": {
+			"type": "object",
+			"description": "Security sub-assessment on a resource",
+			"properties": {
+				"properties": {
+					"x-ms-client-flatten": true,
+					"$ref": "#/definitions/SecuritySubAssessmentProperties"
+				}
+			},
+			"allOf": [{
+				"$ref": "../../../common/v1/types.json#/definitions/Resource"
+			}]
+		},
+		"SecuritySubAssessmentProperties": {
+			"type": "object",
+			"description": "Describes properties of an sub-assessment.",
+			"properties": {
+				"id": {
+					"readOnly": true,
+					"type": "string",
+					"description": "Vulnerability ID."
+				},
+				"displayName": {
+					"readOnly": true,
+					"type": "string",
+					"description": "User friendly display name of the sub-assessment"
+				},
+				"status": {
+					"$ref": "#/definitions/SubAssessmentStatus"
+					
+				},
+				"remediation": {
```

Comment:
>"remediation": { [](start = 4, length = 16)

it would be great if we could provide (aka.ms) link to docs that has details either as part of one of the properties or as separate uri property

Reply:
Not sure I understood this point. "remediation" is data we're getting from the sub-assessment provider and it is a dynamic string instructions on how to remediate the security issue.
Although it could be a url, it won't necessarily be one.